--- Makefile.PL	Wed Aug 28 21:16:33 2002
+++ Makefile.PL.new	Wed Aug 28 21:17:51 2002
@@ -322,7 +322,7 @@
 	'XSOPT'	=> $XSOPT,
     # do not even think about 'INC' => '-I/usr/ucbinclude', Solaris will avenge.
 	'INC'	=> '',     # e.g., '-I/usr/include/other' 
-        'MAN3PODS'  => {},  # Pods will be built by installman.
+#       'MAN3PODS'  => {},  # Pods will be built by installman.
 	'dist'      => {
 	    'CI'       => 'ci -l',
 	    'COMPRESS' => 'gzip -9f', 
